spendsterIOS/CornerButton.swift

```diff
@@ -0,0 +1,98 @@
+//
+//  CornerButton.swift
+//  spendsterIOS
+//
+//  Created by Dmytro Holovko on 3/27/19.
+//  Copyright © 2019 KeyToTech. All rights reserved.
+//
+// swiftlint:disable force_cast
+import Foundation
+import UIKit
+
+@IBDesignable class CornerButton: UIButton {
```

Comment:
@Thongvor please create a "Presentation" group and move all UI files from "spensterIOS" into this sub group.